src/Microsoft.ML.Core/Data/AnnotationUtils.cs

```diff
@@ -441,8 +441,15 @@ public static IEnumerable<SchemaShape.Column> GetTrainerOutputAnnotation(bool is
         public static IEnumerable<SchemaShape.Column> AnnotationsForMulticlassScoreColumn(SchemaShape.Column? labelColumn = null)
         {
             var cols = new List<SchemaShape.Column>();
-            if (labelColumn != null && labelColumn.Value.IsKey && NeedsSlotNames(labelColumn.Value))
-                cols.Add(new SchemaShape.Column(Kinds.SlotNames, SchemaShape.Column.VectorKind.Vector, TextDataViewType.Instance, false));
+            if (labelColumn != null && labelColumn.Value.IsKey)
+
```

Comment:
It would probably be good to wrap this in curly braces, now that it is more than one line (and that you have this blank line in between the if and the body.